coordinator/src/main/scala/filodb.coordinator/queryplanner/MultiPartitionPlanner.scala

```diff
@@ -140,7 +140,7 @@ class MultiPartitionPlanner(partitionLocationProvider: PartitionLocationProvider
       case lp: ApplyAbsentFunction        => super.materializeAbsentFunction(qContext, lp)
       case lp: ScalarBinaryOperation      => super.materializeScalarBinaryOperation(qContext, lp)
       case lp: ApplyLimitFunction         => super.materializeLimitFunction(qContext, lp)
-      case _: TopkCardinalities           => throw new IllegalArgumentException("TopkCardinalities unexpected here")
+      case _: TsCardinalities             => throw new IllegalArgumentException("TsCardinalities unexpected here")
```

Comment:
When we provide just the ws and we need to get all ws/ns under the given ws, it will be a multi partition call. We will need this implementation in that case.

Reply:
deferred to later commit